Extension/src/Debugger/configurationProvider.ts

```diff
@@ -34,10 +38,55 @@ abstract class CppConfigurationProvider implements vscode.DebugConfigurationProv
             return undefined;
         }
 
+        const winDir: string = process.env.WINDIR ? process.env.WINDIR.toLowerCase() : null;
+        const winDirAltDirSep: string =  process.env.WINDIR ? process.env.WINDIR.replace('\\', '/').toLowerCase() : null;
+
+        // Help WSL users with using the correct pipeProgram if the one they selected does not exist.
+        if (os.platform() === 'win32' &&
+            config.pipeTransport &&
```

Comment:
We also probably want to check the instance where pipeProgram isn't rooted and pipeCwd is specified.